integreat_cms/sitemap/sitemaps.py

```diff
@@ -144,9 +144,13 @@ def __init__(self, region, language):
         # Instantiate WebappSitemap
         super().__init__(region, language)
         # Filter queryset based on region and language
-        self.queryset = self.queryset.filter(
-            page__in=self.region.get_pages(return_unrestricted_queryset=True),
-            language=self.language,
+        self.queryset = (
+            self.queryset.filter(
+                page__in=self.region.get_pages(return_unrestricted_queryset=True),
+                language=self.language,
+            )
+            .order_by("page", "-version")
+            .distinct("page")
```

Comment:
```suggestion
            .distinct("page_pk")
```
If you use `page_pk` as distinct value (like in the default ordering), you don't need to order first.

Reply:
But that would list translations in ascending order? And don't we need to get the slug from the latest version?

Reply:
No, translations are ordered by `["page__pk", "language__pk", "-version"]` by default.